notifications.bs

```diff
@@ -471,6 +530,7 @@ interface Notification : EventTarget {
   readonly attribute boolean requireInteraction;
   readonly attribute boolean sticky;
   [SameObject] readonly attribute any data;
+  readonly attribute FrozenArray&lt;NotificationAction> actions;
```

Comment:
Done (indeed, there's currently no reason the UA would replace actions with a different FrozenArray).